cmake/modules/FindSYMENGINE.cmake

```diff
@@ -118,7 +117,22 @@ STRING(REGEX REPLACE
   )
 REMOVE_DUPLICATES(_symengine_include_dirs)
 
-SET(_symengine_libraries ${SYMENGINE_LIBRARIES})
+#
+# Get the full path for the SYMENGINE_LIBRARIES. We could also do this using
+# DEAL_II_FIND_LIBRARY, but since these libraries are CMake targets we can
+# query them directly for this information.
+# 
+FOREACH(SYMENGINE_LIBRARY_NAME ${SYMENGINE_LIBRARIES})
+  GET_PROPERTY(SYMENGINE_LIBRARY TARGET ${SYMENGINE_LIBRARY_NAME} PROPERTY LOCATION)
```

Comment:
```suggestion
   if (TARGET ${SYMENGINE_LIBRARY_NAME})
       GET_PROPERTY(SYMENGINE_LIBRARY TARGET ${SYMENGINE_LIBRARY_NAME} PROPERTY LOCATION)
   else ()
       SET(SYMENGINE_LIBRARY ${SYMENGINE_LIBRARY_NAME})
   endif()
```

Reply:
Thanks. I don't see any entries other than `symengine` appear in `SYMENGINE_LIBRARIES`, so its been hard to sanity check this.

Reply:
You are probably using a SymEngine build with no library dependencies (except LLVM).

Reply:
Yeah, that must be it. I use boostmp for integers, LLVM, and nothing else. I'll check with GMP etc. at a later stage. Thanks for the tip :-)

Reply:
BTW. I wrote all of the CMake stuff about 2 years ago, so a lot has changed since then!